### ISSUE TITLE
1.7 If possible make Void Monster and Skyblock mutually exclusive

### ISSUE THREAD

**Issue by artaficialAI** [reteo/Survival-Industry#147]
Someone who doesn't know any better could have a really bad day if they enable the Void Monster mod and are playing on a skyblock because there would be no bedrock to stop it.

**Comment by reteo** [comment]
Yeah, that makes sense.  I can probably switch the mutual exclusion from MAtmos to Void Monster.

**Comment by reteo** [comment]
Void Monster is now exclusive with Skyblock Game.  Just give it some time for ATLauncher to update.
